routing_model/index.html

```diff
@@ -591,95 +611,109 @@ <h2>Deciding that a member of our section should be relocated away</h2>
   All in all, we have the certainty that eventually, exactly one of these two RPCs will make it to our section.<br/>
-  When it does, it will be voted in PARSEC, regardless of the order of operations (see previous diagram), so it will eventually make it through PARSEC.<br/>
-  If they refused our candidate, our job ends there: the candidate will simply be a candidate for relocation again later, so we will try to relocate it then.<br/>
+  When it does, it will be voted in PARSEC, regardless of the order of operations, so it will eventually make it through PARSEC.<br/>
+  <br/>
+  If a node in RelocatingState is a non-elder adult node, we can try to relocate it. If they are an elder node, we will first wait for the Adult/Elder promotion/demotion flow to kick in and demote them to an adult. This will happen because we changed their state from OnlineState to RelocatingState, which means that they will get demoted.<br/>
+  <br/>
+  On sending ExpectCandidate RPC, we set the node state to WaitRelocateResponseState when so we know after split or merge that we are waiting for response.<br/>
+  If they refused our candidate, our job ends there: We reset the state to RelocatingState and the candidate will simply be a candidate for relocation again later, so we will try to relocate it then.<br/>
   If they accepted the node and sent us a RelocateResponse RPC, so the ParsecRelocateResponse event reached consensus, we will send to the node that is being relocated the RelocatedInfo the will need.<br/>
-  At this point, we may purge their information since this node isn't a member of our section anymore.
+  At this point, we will purge their information since this node isn't a member of our section anymore.
+  <br/>
+  Also of note: While we are preventing OnlineState node from relocating when we are already relocating node, we may be actively relocating more nodes away from our section.<br/>
+  However, we will only do one at a time per CheckRelocateTimeOut event.
   </p>
+  <button class="collapsible">get_best_relocating_node_and_target</button>
+  <div class="content">
+    <p>Returns the best node to relocate and the target address to send it to<br/>
+    There may be mutliple nodes relocating, for example because of a merge. Take the best one (oldest), and choose a target address.<br/>
+    The target address is one managed by one of our neighbour. This could be random, or the current old_public_id with a single bit of the prefix flipped.<br/>
+    This would would help ensure that source and destination remain neighbours, even if the source split.<br/>
```

Comment:
This makes me wonder: what happens if we're relocating a node to a sibling, but then we merge and they get relocated to our own section? (Probably just works, but let's make sure)

Reply:
Yes, there is nothing preventing self relocation.
I'll add a note.